### PR TITLE
clarify source/destination

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#4692

### CODE REVIEW

**Comment by illfated** [comment]
@JohanFreelancer9 : I was about to suggest a couple of minor edits, but I would prefer to have your feedback first.
I am thinking about the following phrases:
```
connection was initiated
connection was attempted
```
(See also the full phrasing in the PR content.)

To my mind, it would be more natural to use the indefinite article and say "a connection was initiated" or "a connection was attempted", but I am also pondering if it would be more precise to use the definite article "the" and say "the connection was initiated" or "the connection was attempted". What do you think will fit best in this scenario?

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/auditing/event-5156.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ This event generates when [Windows Filtering Platform](https://msdn.microsoft.co
 
     -   127.0.0.1 , ::1 - localhost
 
--   **Source Port** \[Type = UnicodeString\]**:** port number on which application received the connection.
+-   **Source Port** \[Type = UnicodeString\]**:** port number on which the connection was initiated.
```
Comment (illfated) [inline windows/security/threat-protection/auditing/event-5156.md]: minor adjustment for standard phrasing:
```suggestion
-   **Source Port** \[Type = UnicodeString\]**:** port number from which the connection was initiated.
```

File: windows/security/threat-protection/auditing/event-5156.md [hunk 2]
```diff
@@ -114,9 +114,9 @@ This event generates when [Windows Filtering Platform](https://msdn.microsoft.co
...
 
--   **Destination Address** \[Type = UnicodeString\]**:** IP address ***from*** which connection was received or initiated.
+-   **Destination Address** \[Type = UnicodeString\]**:** IP address to which the connection was initiated.
```
Comment (illfated) [inline windows/security/threat-protection/auditing/event-5156.md]: phrasing adjustment for more precise wording:
```suggestion
-   **Destination Address** \[Type = UnicodeString\]**:** IP address where the connection was received.
```

File: windows/security/threat-protection/auditing/event-5156.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ This event generates when [Windows Filtering Platform](https://msdn.microsoft.co
 
     -   127.0.0.1 , ::1 - localhost
 
--   **Destination Port** \[Type = UnicodeString\]**:** port number which was used from remote machine to initiate connection.
+-   **Destination Port** \[Type = UnicodeString\]**:** port number on which the connection was initiated.
```
Comment (illfated) [inline windows/security/threat-protection/auditing/event-5156.md]: same again:
```suggestion
-   **Destination Port** \[Type = UnicodeString\]**:** port number where the connection was received.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @illfated , done

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4692

Thank you.

cc: @kenwith

**Pull request merged**